### PR TITLE
Add_TON_prices_ethereum_tokens.sql

### PR DESCRIPTION
**Pull request by kooroot**
Add_TON_prices_ethereum_tokens

### CODE REVIEW

**Comment by kooroot** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by kooroot** [comment]
recheck

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -1626,7 +1626,8 @@ FROM
     ('val-valeria', 'ethereum', 'VAL', 0x011e128ec62840186f4a07e85e3ace28858c5606, 18),
     ('propc-propchain', 'ethereum', 'PROPC', 0x9ff58067bd8d239000010c154c6983a325df138e, 18),
     ('beam-beam-eth', 'ethereum', 'BEAM', 0x62d0a8458ed7719fdaf978fe5929c6d342b0bfce, 18),
-    ('sdl-stakelink', 'ethereum', 'SDL', 0xa95c5ebb86e0de73b4fb8c47a45b792cfea28c23, 18)
+    ('sdl-stakelink', 'ethereum', 'SDL', 0xa95c5ebb86e0de73b4fb8c47a45b792cfea28c23, 18),
+    ('tokamak-network-token', 'ethereum', 'TON', 0x2be5e8c109e2197D077D13A82dAead6a9b3433C5, 18)
```
Comment (jeff-dude) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1630]: @kooroot it looks like this token_id doesn't exist on the pricing provider:
https://github.com/duneanalytics/spellbook/actions/runs/6781296545/job/18431356138?pr=4732#step:6:32

https://api.coinpaprika.com/v1/coins/tokamak-network-token

Comment (kooroot) [reply models/prices/ethereum/prices_ethereum_tokens.sql:1630]: @jeff-dude Hi, jeff
I search coinpaprika it seem https://coinpaprika.com/ko/coin/ton-tokamak-network/
the pricing provider:
https://api.coinpaprika.com/v1/coins/ton-tokamak-network/

I will modified and submit PR.

Comment (kooroot) [reply models/prices/ethereum/prices_ethereum_tokens.sql:1630]: I modified and submit PR. https://github.com/duneanalytics/spellbook/pull/4751

Comment (Hosuke) [reply models/prices/ethereum/prices_ethereum_tokens.sql:1630]: After #4751 is merged, we may close this PR.

**Pull request closed without merging**